lib/preprocess.js

```diff
@@ -143,13 +143,18 @@ export const fetchMeasurements = async cid => {
       for await (const buf of entry.content()) {
         bufs.push(buf)
       }
-      const json = Buffer.concat(bufs).toString()
-      return JSON.parse(json)
+      return parseMeasurements(Buffer.concat(bufs).toString())
     }
   }
   throw new Error('No measurements found')
 }
 
+export const parseMeasurements = str => {
+  const ret = str.split('\n').filter(Boolean).map(line => JSON.parse(line))
```

Comment:
This assumes the old JSON format was never pretty-printed, i.e. the array was on a single line.

I guess it's an okay assumption, but I think it's worth documenting it here. Could you please add a code comment?